csharp/src/Google.Protobuf/JsonFormatter.cs

```diff
@@ -842,13 +911,17 @@ public Settings(bool formatDefaultValues, TypeRegistry typeRegistry) : this(form
             /// <param name="formatDefaultValues"><c>true</c> if default values (0, empty strings etc) should be formatted; <c>false</c> otherwise.</param>
             /// <param name="typeRegistry">The <see cref="TypeRegistry"/> to use when formatting <see cref="Any"/> messages. TypeRegistry.Empty will be used if it is null.</param>
             /// <param name="formatEnumsAsIntegers"><c>true</c> to format the enums as integers; <c>false</c> to format enums as enum names.</param>
+            /// <param name="indent">The indentation string to use for multi-line formatting. <c>""</c> to disable multiline format.</param>
```

Comment:
Any reason to prefer "" in both the docs and the default here? I'd prefer null as a clearer indication of "I don't want indentation" than an empty string. (And if you go with the first of the options above, coalescing empty string to null for the Indent/Indentation property, I'd definitely use null here.)

Reply:
Nit: if multi-line is going to be hyphenated, do it in both places in the line :)